nestedtensor/nested/nested.py

```diff
@@ -213,11 +213,104 @@ def nested_stride(self, dim=None):
     def __torch_function__(self, func, types, args=(), kwargs=None):
         _local_func = None
         if func in NestedTensor.__C_functions:
+            if 'interpolate' in str(func):
+                # size can be None, torch.Size, int or tuple of int
+                if kwargs['size'] is None:
+                    size = kwargs['size']
+                elif isinstance(kwargs['size'], tuple) or isinstance(kwargs['size'], torch.Size):
```

Comment:
You can enable overloads by registering multiple pybind functions under the same number but with different arguments. pybind infers the signatures. overload resolution is done in order of registration.